index.html

```diff
@@ -42,5 +43,185 @@ <h2 class="Supportingtext Title2">I'm a Software Developer</h2>
+          <ul class="list_feat list_card_gral">
+            <li class="li_btn_2">HTML</li>
+            <li class="li_btn_2">Bootstrap</li>
+            <li class="li_btn_2">Ruby</li>
+          </ul>
+        </div> 
+        <a href="#" class="project_btn_all project_btn_2">See Project</a>
+      </div>
+    </div>
+    <div class="card_cnt work-6">
+      <div class="info_card">
+        <div class="cnt_info_card">
+          <h4 class="Title1_card">Pet Shopping Store</h4>
+          <p class="Title2_card">Here you can find the best productos for your pet</p>
+          <ul class="list_feat list_card_gral">
+            <li class="li_btn_2">HTML</li>
+            <li class="li_btn_2">Bootstrap</li>
+            <li class="li_btn_2">Ruby</li>
+          </ul>
+        </div> 
+        <a href="#" class="project_btn_all project_btn_2">See Project</a>
+      </div>
+    </div>
+  </div>
+
+  </section>
+
+  <!-- Section About -->
+  <section class="about back_dark">        
+    <div class="about_container">
+      <div class="about_text">
+        <h3 class="Title1 Tittle_about">About me</h3>
+        <h4 class="Simpletext">Hello I'm a software developer! I can help you build a product, feauture or website. Look through some of my work and expererience!. If you like what you see and have a prject you need coded, don't hesitate to contact me.</h4>
+        <a class="project_btn_all project_btn_1 resume_btn" href="">Get My Resume</a>
+      </div>            
+    </div>  
+  </section>
+  
+  <!-- Section Skills -->
+  <section class="back_dark">
+    <span class="line_div_dark"></span>         
+    <div class="skills">            
+      <ul class="ul_skill">
+        <li class="li_skills Title2"><h4><img src="/images/diamond_icon.png" alt="Diamond icon">Languages</h4></li>
+        <li class="li_skills Simpletext">JavaScript</li>
+        <li class="li_skills Simpletext">Ruby</li>
+        <li class="li_skills Simpletext">HTML</li>
+        <li class="li_skills Simpletext">CSS</li>
+      </ul>             
+      <ul class="ul_skill">
+        <li class="li_skills Title2"><h4><img src="/images/rectangle_icon.png" alt="Rectangle icon">Frameworks</h4></li>
+        <li class="li_skills Simpletext">React.js</li>
+        <li class="li_skills Simpletext">Ruby on Rails</li>
+        <li class="li_skills Simpletext">RS Pec</li>
+        <li class="li_skills Simpletext">CapyBara</li>
+      </ul>            
+      <ul class="ul_skill">
+        <li class="li_skills Title2"><h4><img src="/images/circle_icon.png" alt="circle icon">Skills</h4></li>
+        <li Class="li_skills Simpletext">Database Managment</li>
+        <li Class="li_skills Simpletext">Version Control</li>
+        <li Class="li_skills Simpletext">CLI</li>
+        <li Class="li_skills Simpletext">API Design</li>
+        <li Class="li_skills Simpletext">Web Development</li>
+      </ul>
+    </div>               
+  </section>
```

Comment:
Please consider following these 2 rules in this section as it's a good practice for semantics:-

1. The technology categories ("Languages", "Frameworks" and "Skills") are in a `ul`, each of them in a `li`
2. The technology in each category(e.g., "JavasScript", "HTML" and "CSS") are also a list `ul` and each of them a `li`


## Solution with detailed demonstration:-

Following is the code representation of the above if you are confused about what the instructions mean:-


```
    <ul>
      <li>
        Languages
        <ul>
          <li>JavaScript</li>
          <li>Ruby</li>
          <li>Html</li>
          <li>CSS</li>
        </ul>
      </li>
      <li>
        Frameworks
        <ul>
          <li>Bootstrap</li>
          <li>Ruby on Rails</li>
          <li>Rspec</li>
          <li>Capybara</li>
          <li>Selenium</li>
        </ul>
      </li>
      <li>
        Skills
        <ul>
          <li>Database Management</li>
          <li>Version Control</li>
          <li>CLI</li>
          <li>API Design</li>
          <li>Web Development</li>
        </ul>
      </li>
    </ul>
```

Reply:
Done, Thank you @ABDULALI3468